src/EventListener/AddContentLanguageHeaderListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class AddContentLanguageHeaderListener implements EventSubscriberInterface
+{
+    public function onKernelResponse(ResponseEvent $event)
+    {
+        if (HttpKernelInterface::MAIN_REQUEST != $event->getRequestType()) {
+            return;
+        }
+
+        $request = $event->getRequest();
+        /** @var \eZ\Publish\API\Repository\Values\Content\Content $content */
+        $content = $request->attributes->get('content');
+        if ($content) {
+            $contentLanguage = $content->prioritizedFieldLanguageCode ?? $content->versionInfo->initialLanguageCode;
```

Comment:
This is Dependency Inversion Principle violation. `$content->prioritizedFieldLanguageCode` is not part of the PAPI. Moreover, it's marked as internal.

Reply:
Also `VersionInfo::$initialLanguageCode` is not required and can be `null` in some cases (i.e. when creating content through PAPI).